Add content/pt-br/tightly_coupled_architectures.md (#783)
cncf/glossary: content/pt-br/tightly_coupled_architectures.md

---
title: Arquitetura Acoplada
status: Completed

> category: Propriedade

Reviewer: ```suggestion
category: propriedade
```

Author: Hi, it's ok for me, but I repeated the capital letter in original text.

---

> Arquitetura acoplada é um estilo de arquitetura em que vários componentes da aplicação são interdependentes (o paradigma oposto de [arquiteturas desacopladas](/loosely_coupled_architecture/)). Isso significa que uma mudança em um componente provavelmente afetará outros componentes. Geralmente é mais fácil de implementar do que estilos de arquitetura desacopladas, mas pode deixar um sistema mais vulnerável a falhas em cascata. Eles também tendem a exigir implementações coordenadas de componentes, o que pode se tornar um obstáculo à produtividade do desenvolvedor.

Reviewer: ```suggestion
Arquitetura acoplada é um estilo de arquitetura em que vários componentes da aplicação são interdependentes (o paradigma oposto de [arquiteturas desacopladas](/loosely_coupled_architecture/)). Isso significa que uma mudança em um componente provavelmente afetará outros componentes. Geralmente é mais fácil de implementar do que estilos de arquitetura desacopladas, mas pode deixar um sistema mais vulnerável a falhas em cascata. Esse estilo de arquitetura também tende a exigir implementações coordenadas de componentes, o que pode se tornar um obstáculo à produtividade do desenvolvedor.
```

> Arquiteturas acopladas de aplicativos são uma maneira bastante tradicional de criar aplicações. Embora não sejam necessariamente consistentes com todas as melhores práticas de desenvolvimento de [microsserviços](/microservices/), elas podem ser úteis em circunstâncias específicas. Eles tendem a ser mais rápidos e simples de implementar e, assim como [aplicações monolíticas](/monolithic_apps/), eles podem acelerar o ciclo de desenvolvimento inicial.

Reviewer: ```suggestion
Arquiteturas acopladas de aplicativos são uma maneira bastante tradicional de criar aplicações. Embora não sejam necessariamente consistentes com todas as melhores práticas de desenvolvimento de [microsserviços](/microservices/), elas podem ser úteis em circunstâncias específicas. Elas tendem a ser mais rápidas e simples de implementar e, assim como [aplicações monolíticas](/monolithic_apps/), elas podem acelerar o ciclo de desenvolvimento inicial.
```
